cli/src/semdep/parse_lockfile.py

```diff
@@ -21,7 +23,7 @@
 
 logger = getLogger(__name__)
 
-from semgrep.semgrep_interfaces.semgrep_output_v0 import FoundDependency
+from semgrep.semgrep_interfaces.semgrep_output_v0 import FoundDependency, Transitivity
```

Comment:
You import Transitivity twice

Reply:
I also cant find these Classes in the atd repo

Reply:
I need to make a PR there